interface-definitions/protocols_openfabric.xml.in

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="protocols">
+    <children>
+      <node name="openfabric" owner="${vyos_conf_scripts_dir}/protocols_openfabric.py">
+        <properties>
+          <help>OpenFabric protocol</help>
+          <priority>680</priority>
+        </properties>
+        <children>
+          <tagNode name="interface">
+            <properties>
+              <help>Interface params</help>
+              <completionHelp>
+                <script>${vyos_completion_dir}/list_interfaces</script>
+              </completionHelp>
+              <valueHelp>
+                <format>txt</format>
+                <description>Interface name</description>
+              </valueHelp>
+              <constraint>
+                #include <include/constraint/interface-name.xml.i>
+              </constraint>
+            </properties>
+            <children>
+              <leafNode name="csnp-interval">
+                <properties>
+                  <help>Set Complete Sequence Number Packets (CSNP) interval</help>
```

Comment:
We don't need the word "Set" here, since it's an option name and that name is a noun.

(I suggested to out a verb "Use" above in authentication type options because those options are behavior modifiers, not options with parameters, but even that is subjective. The general guidelines is to avoid verbs in help strings, unless it's really better with a verb.)